StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethodFormFactory/SavedPaymentMethodFormFactory+Link.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SavedPaymentMethodFormFactory+Link.swift
+//  StripePaymentSheet
+//
+//  Created by Till Hellmund on 4/26/25.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+extension SavedPaymentMethodFormFactory {
+    func makeLink(configuration: UpdatePaymentMethodViewController.Configuration) -> PaymentMethodElement {
```

Comment:
We basically just want a read-only form so that the screen doesn't look empty. The only real action that the user can take is deleting the payment method.

Reply:
Can we add snapshot tests for this?

Reply:
Done!